en/news/_posts/2019-04-23-move-to-git-from-svn.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: news_post
+title: "Move to Git from Subversion"
+author: "hsbt"
+translator:
+date: 2019-04-23 00:00:00 +0000
+lang: en
+---
+
+Today, The canonical repository of the Ruby language was moved to Git from Subversion.
+
+The new repository is https://git.ruby-lang.org built by cgit. We can keep your commit hash on the ruby repository directly.
```

Comment:
```suggestion
The web interface for the new repository is [https://git.ruby-lang.org](https://git.ruby-lang.org), and is provided by cgit. We can keep your commit hash on the Ruby repository directly.
```

_note: the given URL is not the git repository itself, therefore the suggested change_

The second sentence is rather unclear: „can“ or „will“?, who is „you“? (every post reader??? ... possibly the existing committers?), what‘s the „commit hash“? ...